query-node/substrate-query-framework/index-builder/src/db/entities.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class EventHistory {
```

Comment:
This name is not very good, really this entity represents that entity was saved to the database as a result of processing an event. So a better name could be `SavedEntityEvent`.